moto/events/models.py

```diff
@@ -338,13 +340,17 @@ def put_targets(self, name, targets):
     def put_events(self, events):
         num_events = len(events)
 
+        response = {"FailedEntryCount": 0, "Entries": []}
```

Comment:
The format of the response should still be the responsibilty of responses.py.
Maybe this method can simply return a tuple with the required information: `return (failed_entries, entries_list)`

Reply:
Hi @bblommers thanks for the review. 

I based this implementation on the way the kinesis `put_events` worked. I think the reason that this response is generated in the models is because the order of the events needs to be retained based on the way they came in. If we were to return a tuple, we'd need some way to re-combine the tuple into the original order. I can definitely do that, but I think it adds an unnecessary level of complexity in the implementation.

If you still feel strongly that the response shouldn't be generated here, just let me know and I'll come up with an implementation. I just wanted you to know the reasoning why I implemented it this way.

Reply:
Here is an alternate implementation, let me know if you like this better.

**responses.py**
```python
def put_events(self):
        events = self._get_param("Entries")

        entries = self.events_backend.put_events(events)

        failed_count = len([e for e in entries if "ErrorCode" in e])
        response = {
            "FailedEntryCount": failed_count,
            "Entries": entries,
        }

        return json.dumps(response)
```

**models.py**

```python
def put_events(self, events):
        num_events = len(events)

        if num_events < 1:
            raise JsonRESTError("ValidationError", "Need at least 1 event")
        elif num_events > 10:
            raise JsonRESTError("ValidationError", "Can only submit 10 events at once")

        # We dont really need to store the events yet
        return [{"EventId": uuid4()} for _ in events]
```

Reply:
Much better @loganasherjones - I'd be happy with that

Reply:
pushed!